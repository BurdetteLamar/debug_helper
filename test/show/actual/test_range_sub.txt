---
DebugHelperTest::RangeSub (message='test_range_sub'):
  first: 0
  last: 4
  exclude_end?: false
