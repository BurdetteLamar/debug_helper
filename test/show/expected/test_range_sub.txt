---
DebugHelperTest::RangeSub (message='test_range_sub'):
  DebugHelperTest::RangeSub#first: 0
  DebugHelperTest::RangeSub#last: 4
  DebugHelperTest::RangeSub#exclude_end?: false
